main.tex

```diff
@@ -809,6 +809,138 @@ \subsection{Inf-sup conditions}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Saddle point problems}
+
+A saddle point problem has the form:
+\begin{displaymath}
+    \begin{bmatrix}
+        A & B^{T}\\ 
+        B & 0
+    \end{bmatrix}
+    \begin{bmatrix}
+        u \\ p
+    \end{bmatrix}
+    =
+    \begin{bmatrix}
+        f \\ g
+    \end{bmatrix}.
+\end{displaymath}
+
+To grasp the importance of this formulations, let's first see
+some examples:
+\begin{itemize}
+    \item \textbf{Stokes}: The variational formulation of
+    Stokes reads:
+    \begin{displaymath}
+    \begin{cases}
+        (\nabla u, \nabla v) - (p, \dive v)
+        &=
+        \langle f, v \rangle
+        \quad\forall v
+        \\
+        (\dive u, q) &= 0 \hspace{3em}\forall q
+    \end{cases}
+    \end{displaymath}
+    From where it's clear that \(A=-\Delta\), \(B=\dive\).
+
+    \item \textbf{Darcy (or mixed Poisson)}: We are dealing 
+    with the problem
+    \begin{displaymath}
+        -\Delta u = f
+    \end{displaymath}
+    Introducing the variable \(\sigma = -\nabla u\), the problem
+    now reads:
+    \begin{displaymath}
+    \begin{cases}
+        \dive \sigma &= f \\ 
+        \dive \sigma + \nabla u &= 0
+    \end{cases}
+    \end{displaymath}
+    Acting the equation over test function \(\tau\) and 
+    \(v\) we end up with the saddle point problem. 
+    \begin{displaymath}
+    \begin{cases}
+        (\sigma, \tau) - (u, \dive \tau) &= 0
+        \hspace{3em} \forall \tau \\ 
+        (\dive \sigma, v) &= \langle f,v \rangle
+        \hspace{1.2em} \forall v
+    \end{cases}
+    \end{displaymath}
+
+    \item \textbf{Primal-Mixed Poisson (Dirichlet with multipliers)}
+    Consider the problem
+    \begin{displaymath}
+    \begin{cases}
+        -\Delta u &= f \text{ over } \Omega \\ 
```

Comment:
no estoy muy seguro de 'over'. En general, el estandar es usar 'in' Omega y después 'on' la frontera.